components/textbook-demo/LearningPathPagesSection.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <section class="learning-path-pages-section">
+    <div>
+      <h2 class="copy__title">
+        Learning path pages
+      </h2>
+      <nav class="learning-path-pages-section__nav">
+        <ul class="learning-paths">
+          <li
+            v-for="{ label, progress, url } in learningPaths"
+            :key="label"
+            class="learning-path"
+          >
+            <div class="learning-path__connection" />
+            <ProgressCircle :progress="progress" />
+            <a
```

Comment:
**BasicLink** doesn't support the `@mouseover` directive, which would break the _hover to select_ functionality. 😢

We could start extending **BasicLink** and map directives, but I'm not sure that would be the best approach here because:

- That directive isn't used anywhere else, so it would make the component unnecessarily more complex
- This functionality is just for the demo, so there is a chance that it would make it to the final codebase
- BasicLink is a wrapper on a dynamic component, which would make these kind of _directive injection_ somewhat unpredictable in behaviour

What do you suggest we do here? 🤔